movement/io/load_poses.py

```diff
@@ -109,7 +111,7 @@ def from_sleap_file(
     preferred format for loading pose tracks from SLEAP into *movement*.
 
     You can also try directly loading the ".slp" file, but this feature is
-    experimental and doesnot work in all cases. If the ".slp" file contains
+    experimental and does not work in all cases. If the ".slp" file contains
     both user-labeled and predicted instances, only the predicted ones will be
     loaded. If there are multiple videos in the file, only the first one will
     be used.
```

Comment:
I think this section of the docstring could be updated. This is the user's enty point for SLEAP files, so we want to make sure this docstring is solid.

```suggestion
    You can also try directly loading the ".slp" file, but this feature
    does not work in all cases. If the ".slp" file contains only predicted
    intances, they will be imported alongside their point-wise confidence scores.
    Alternatively, if it contains only user-labeled instances, they will be
    loaded and assigned fixed point-wise confidence scores of 1.0.
    Lastly, if a mix of both user-labeled and predicted instances are present,
    only the predicted ones will be loaded.
    If there are multiple videos in the file, only the first one will be used.
```

Feel free to edit if my interpretation of the expected behaviour is incorrect